src/ZkFold/Symbolic/Data/Helpers.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE TypeOperators       #-}
+module ZkFold.Symbolic.Data.Helpers where
+
+import           Data.Constraint
+import           Data.Constraint.Nat
+import           Data.Constraint.Unsafe           (unsafeAxiom)
+import           Prelude
+
+import           ZkFold.Base.Algebra.Basic.Number
+
+
+-- type ExtensionBits inputLen = 8 * (128 - Mod inputLen 128)
+-- type ExtendedInputByteString inputLen c = ByteString (8 * inputLen + ExtensionBits inputLen) c
+
+
+with4n6 :: forall n {r}. KnownNat n => (KnownNat (4 * n + 6) => r) -> r
```

Comment:
As this is used in Plonk code, it's better to move it there. Maybe near `PlonkupPolyExtendedLength` definition is a good place.